### DISCUSSION TITLE
Fresh nuxt project does not work

### DISCUSSION THREAD

**Discussion by lkrombeen** [nuxt/nuxt#28538] [Questions]
Hello all,

I have created a new Nuxt project using npm and node v20.16.0:
```
npx nuxi@latest init app
cd app
npm run dev
```

At first glance this seemed ok. The Welcome app was showing, except that I got a warning in my Chrome console:
```
Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/html". Strict MIME type checking is enforced for module scripts per HTML spec.
```

I decided to ignore this and continue with a simple button that increments a number. This was done in a new page. The click action did not seem to work. Testing the code in a random sandbox environment works. I think it might be because of the following error I am constantly getting:
```
No match found for location with path "/_nuxt/C:/Users/<user>/Projects/nuxt-test/app/node_modules/nuxt/dist/app/entry.js"
```
**edit**: If I only use app.vue and put the button in there it also does not work but I get a different warning:
`Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/html". Strict MIME type checking is enforced for module scripts per HTML spec.`

Below is my code.

app.vue:
```
<template>
  <div>
    <NuxtPage />
  </div>
</template>
```

pages/index.vue
```
<template>
  <div>
    <h1>{{ count }}</h1>
    <button @click="increment">Increment</button>
  </div>
</template>

<script setup lang="ts">
const count = ref(0)
const increment = () => count.value++
</script>
```

This code works in an existing repository, but it does not seem to work in my fresh Nuxt project. What am I missing?

Steps to reproduce:
1. Install Node v20.16.0
2. Run `npx nuxi@latest init app` and select npm
3. `cd app`
4. `npm run dev`
5. Change the app.vue as shown above
6. Add pages/index.vue

**Comment by xjccc** [comment]
I create a new nuxt project. I don't reproduction this.
You could use `npx nuxi info` show some project info.
```
- Operating System: Darwin
- Node Version:     v20.10.0
- Nuxt Version:     3.12.4
- CLI Version:      3.12.0
- Nitro Version:    -
- Package Manager:  pnpm@9.5.0
- Builder:          -
- User Config:      compatibilityDate, devtools
- Runtime Modules:  -
- Build Modules:    -
```

**Reply by lkrombeen** [reply]
Thanks for testing it. I have the following:
```
- Operating System: Windows_NT
- Node Version:     v20.16.0
- Nuxt Version:     3.12.4
- CLI Version:      3.12.0
- Nitro Version:    2.9.7
- Package Manager:  npm@10.8.1
- Builder:          -
- User Config:      compatibilityDate, devtools
- Runtime Modules:  -
- Build Modules:    -
```

I also have it working on https://stackblitz.com/edit/github-hjdg4x?file=app.vue when I put in the template and script, but on my Windows 10 device it is not working. I also tested it on a different Windows 10 machine where I got the same results

**Reply by xjccc** [reply]
Sorry for that.. I'm not Windows.. Maybe something diffrent on Windows System

**Comment by lkrombeen** [comment]
Also tested it on Ubuntu and no problems there. Seems to be a problem with a path somewhere in Nuxt or the dependencies they use. Created a bug for it: https://github.com/nuxt/nuxt/issues/28541
